app/controllers/testonly/TestFeatureSwitchManagerController.scala

```diff
@@ -31,59 +32,101 @@ class TestFeatureSwitchManagerController @Inject()(
                                                     adminFeatureSwitchConnector: PensionAdministratorFeatureSwitchConnectorImpl
                                                   )(implicit val ec: ExecutionContext) extends FrontendController {
 
+  private def onOrOff(state:Option[Boolean]):String = state match {
```

Comment:
Logic fails when the toggle state is false

Reply:
Sorted